packages/patterns/src/types.js

```diff
@@ -143,61 +143,17 @@ export {};
  */
 
 /**
- * @typedef {RankCompare} FullCompare
```

Comment:
Are static types considered part of the API? This `FullCompare` type is exported, right? So is removing it from this package a breaking change?

Reply:
Hmm, that's a good question. I've updated to reëxport FullCompare to preserve any remaining imports, on the assumption that dropping it _would_ be considered breaking.